src/main/java/Duke.java

```diff
@@ -1,10 +1,136 @@
+import java.lang.reflect.Array;
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+//        System.out.println("Hello from\n" + logo);
+
+//        Task[] list = new Task[100];
+        ArrayList<Task> list = new ArrayList<>();
+        int i = 0;
+        boolean loop = true;
+        System.out.println("Hello! Duke here, how can I help you?");
+        Scanner sc = new Scanner(System.in);
+
+        while (loop) {
```

Comment:
I think you should name your boolean variable as `isSomething` to represent it is actually a boolean. You can refer to the naming convention provided in the text book.